package.json

```diff
@@ -1,83 +1,85 @@
-        "karma": "1.7.1",
-        "karma-chrome-launcher": "2.2.0",
-        "karma-coverage": "1.1.1",
-        "karma-tap": "3.1.1",
-        "karma-tap-pretty-reporter": "3.0.5",
-        "karma-webpack": "2.0.4",
-        "kw-doc": "1.1.1",
-        "kw-web-suite": "5.0.0",
-        "resemblejs": "2.2.6",
-        "tap-markdown": "1.2.1",
-        "tap-spec": "4.1.1",
-        "tape": "4.8.0",
-        "tape-catch": "1.0.6",
-        "xml2js": "0.4.19"
-    },
-    "scripts": {
-        "size": "size-limit --why --config ./webpack.config.js",
-        "validate": "prettier --config ./prettier.config.js --list-different \"Sources/**/*.js\" \"Examples/**/*.js\"",
-        "reformat": "prettier --config ./prettier.config.js --write \"Sources/**/*.js\" \"Examples/**/*.js\"",
-        "reformat-only": "prettier --single-quote --trailing-comma es5 --print-width 80 --arrow-parens always --write",
-        "doc": "kw-doc -c ./Documentation/config.js",
-        "doc:www": "npm t -- --single-run && kw-doc -c ./Documentation/config.js -s",
-        "doc:publish": "kw-doc -c ./Documentation/config.js -mp",
-        "example": "node ./Utilities/ExampleRunner/example-runner-cli.js -c ./Documentation/config.js",
-        "build": "webpack --progress --colors",
-        "build:debug": "webpack --progress --colors --display-modules",
-        "build:release": "npm run build -- -p",
-        "test": "karma start ./karma.conf.js",
-        "test:travis": "karma start ./karma.conf.js --browsers ChromeHeadlessNoSandbox --single-run",
-        "test:debug": "karma start ./karma.conf.js --no-single-run",
-        "commit": "git cz",
-        "semantic-release": "semantic-release"
-    },
-    "config": {
-        "commitizen": {
-            "path": "node_modules/cz-conventional-changelog"
-        }
-    },
-    "size-limit": [
-        {
-            "path": "./Sources/index.js",
-            "limit": "200 KB"
-        }
-    ],
-    "bin": {
-        "vtkDataConverter": "./Utilities/DataGenerator/convert-cli.js",
-        "xml2json": "./Utilities/XMLConverter/xml2json-cli.js"
-    },
-    "nyc": {
-        "include": [
-            "Sources/**/*.js"
-        ],
-        "exclude": [
-            "Sources/**/test/*.js"
-        ]
+  "name": "vtk.js",
+  "version": "0.0.0-semantically-release",
+  "description": "Visualization Toolkit for the Web",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/kitware/vtk-js.git"
+  },
+  "license": "BSD-3-Clause",
+  "bugs": {
+    "url": "https://github.com/kitware/vtk-js/issues"
+  },
+  "homepage": "https://github.com/kitware/vtk-js#readme",
+  "main": "./dist/vtk.js",
+  "dependencies": {
+    "base64-js": "1.2.1",
+    "blueimp-md5": "2.10.0",
+    "commander": "2.11.0",
+    "gl-matrix": "2.3.1",
+    "jszip": "3.1.4",
+    "pako": "1.0.6",
+    "seedrandom": "2.4.3",
+    "shelljs": "0.7.8",
+    "webworker-promise": "^0.4.1",
+    "worker-loader": "^1.1.0"
```

Comment:
This is already provided inside kw-web-suite.

Reply:
Good to know -- I will see if we can build without it.